packages/client/src/raffles/types/patchRaffleParticipation.types.ts

```diff
@@ -0,0 +1,16 @@
+import type { Config } from '../..';
+import type { Raffle, RaffleParticipation } from './raffles.types';
+
+export type PatchRaffleParticipationData = {
+  value: string;
+  path: string;
+  op: string;
+  from?: string;
+};
+
+export type PatchRaffleParticipation = (
+  raffleId: Raffle['id'],
+  participationId: RaffleParticipation['id'],
+  data?: PatchRaffleParticipationData[] | null,
```

Comment:
Shouldn't this be required? If so, remove the `null` as well.

Reply:
Right. I already update it.